routes/web.php

```diff
@@ -171,11 +169,15 @@
     Route::get('/secretariat/registrations/reject/{id}', [GuestsController::class, 'reject'])->name('secretariat.registrations.reject');
     Route::post('/secretariat/invite', [InvitationController::class, 'store'])->name('secretariat.invite');
 
-    /** Application handling */
-    Route::post('/applications/period/update', [ApplicationController::class, 'updateApplicationPeriod'])->name('applications.period.update');
-    Route::get('/applications', [ApplicationController::class, 'showApplications'])->name('applications');
-    Route::post('/applications', [ApplicationController::class, 'editApplication'])->name('applications.edit');
-    Route::get('/applications/export', [ApplicationController::class, 'exportApplications'])->name('applications.export');
+    /** Admission and applicants handling */
+    Route::post('/admission/period/update', [AdmissionController::class, 'updateApplicationPeriod'])->name('admission.period.update');
+    Route::get('/admission/export', [AdmissionController::class, 'export'])->name('admission.export');
+    Route::post('/admission/finalize', [AdmissionController::class, 'finalize'])->name('admission.finalize');
+
+    Route::get('/applicants', [AdmissionController::class, 'index'])->name('admission.applicants.index');
+    Route::get('/applicants/{application}', [AdmissionController::class, 'show'])->name('admission.applicants.show');
+    Route::get('/applicants/{application}/edit', [AdmissionController::class, 'edit'])->name('admission.applicants.edit');
```

Comment:
This seems to be a POST request for me.

Reply:
That function does not even exist, I will delete the route.